src/main/java/uk/gov/hmcts/reform/sendletter/services/ftp/FtpClient.java

```diff
@@ -131,6 +131,36 @@ public void deleteReport(String reportPath) {
         });
     }
 
+    public List<FileInfo> listLetters(String serviceFolder) {
+        return runWith(sftp -> {
+            try {
+                String path = String.join("/", configProperties.getTargetFolder(), serviceFolder);
```

Comment:
Do we need to worry about smth similar as in upload method:
```java
String folder = isSmokeTestFile
    ? configProperties.getSmokeTestTargetFolder()
    : String.join("/", configProperties.getTargetFolder(), serviceFolder);

String path = String.join("/", folder, file.getName());
```

Reply:
smoke test letter will be handled by a separately configured job (as we will want to delete from prod as well)